options/locale/locale_en-US.ini

```diff
@@ -170,7 +170,7 @@ string.desc = Z - A
 
 [error]
 occurred = An error occurred
-report_message = If believe that this is a Gitea bug, please search for issues on <a href="https://github.com/go-gitea/gitea/issues" target="_blank">GitHub</a> or open a new issue if necessary.
+report_message = If you believe that this is a Gitea bug, please search for issues on <a href="%[1]s" target="_blank">GitHub</a> or open a new issue if necessary.
```

Comment:
Of course, if we extract out the link, we need to fix it as well in
```go
templates/status/500.tmpl
45:                                     {{if .SignedUser.IsAdmin}}<p>{{.locale.Tr "error.report_message" | Str2html}}</p>{{end}}
```

Reply:
I am not sure how I can approach this best.

Reply:
I'd guess by replacing the line with
```tmpl
                                     {{if .SignedUser.IsAdmin}}<p>{{.locale.Tr "error.report_message" "https://github.com/go-gitea/gitea/issues" | Str2html}}</p>{{end}}
```

Reply:
Sorry, overcomplicated things and started thinking about whether this should be a config option or whatever. Yeah, makes sense.